envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -216,6 +216,10 @@ message HttpConnectionManager {
     // XFCC header comma separated from other values with the value Cert="PEM".
     // Defaults to false.
     bool cert = 3;
+
+    // [#not-implemented-hide:] Whether to forward the DNS type SAN entries of the client cert.
```

Comment:
DNS type Subject Alternative Name

Reply:
Also, please change SAN to URI type Subject Alternative Name in the comment on top of `san`.